src/helpers/utils.ts

```diff
@@ -32,3 +32,7 @@ export function percentFormatter(value: number): string {
 export const unwrap = (wrappedObject: any) => {
     return wrappedObject[Object.keys(wrappedObject)[0]];
 };
+
+export function stringToFloat(text: string): number {
+    return parseFloat(text);
```

Comment:
Please handle `NaN` case here

Reply:
I checked it with `invariant` function.
Is that ok?

Reply:
Please check my new comment :)